ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/_cq_dialog/.content.xml

```diff
@@ -105,6 +105,31 @@
                                             fieldLabel="ZIP File Name"
                                             name="./zipFileName"
                                             required="{Boolean}true"/>
+                                    <heading
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/heading"
+                                        level="{Long}4"
+                                        text="Maximum Content Size for Downloads"/>
+                                    <well
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/well">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <max-content-size
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                fieldDescription="Sets a maximum size limit for downloads (in bytes). Recommended to match size in AssetDownloadServlet OSGi config. Leave blank for no maximum content size."
+                                                fieldLabel="Max Content Size Limit (bytes)"
```

Comment:
Ideally we pull this value from the AssetDownloadServlet configuration... if this is impossible, it would make more sense to collect the size from authors in MB though I think.

Reply:
sure, we can do that (collect the size in MB) my thought was that since the AssetDownloadServlet collects the size limit in bytes it was more consistent this way...but can make the change...

Reply:
Yeh - figured that was why. Hopefully we can get rid of this entirely by reading straight from the servlets conf.